code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -322,7 +322,7 @@ GLOBAL_LIST_INIT(brass_recipes, list ( \
 	desc = "Sheets made out of brass."
 	singular_name = "brass sheet"
 	icon_state = "sheet-brass"
-	icon = 'icons/obj/items_and_weapons.dmi'
+	icon = 'icons/obj/stack_objects.dmi'
```

Comment:
this should be inherited, so it should thus be removable

Reply:
It is not inherited, tiles use `tiles.dmi`